andela/travel_tool_back
# #160820749 Email notification on approval

#### What does this PR do?
- Sends email notification to requesters when their requests are approved or rejected

#### Description of Task to be completed?
- Create email template
- Leverage on sendMail method created to send email notification to requesters

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_back.git
- Enter into the project directory - cd travel_tool_back
- Rollback and run database migrations - yarn db:rollmigrate
- Install dependencies and start the server - make start
- Start the frontend.
- Make a request and choose yourself as the manager
- Go to Approvals page and approve/reject the just created request
- Check your gmail account for a new message from travela regarding your approved/rejected request
- Click on the view request button to redirect to the request

#### Any background context you want to provide?
- In other to test this feature, you need to update your role on the Users table to that of a manager.
Manager roleId: 50319
- Check the env.example to check the variable that is required to implement this task which is SENDGRID_API_KEY, REACT_APP_AUTH_REDIRECT_URL and APP_EMAIL

#### What are the relevant pivotal tracker stories?
[#160820749](https://www.pivotaltracker.com/story/show/160820749)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-04 at 11 14 50 am" src="https://user-images.githubusercontent.com/23608060/46468433-53e98680-c7c8-11e8-9edb-6f0c47c95c36.png">

#### Questions:
N/A
